### ISSUE TITLE
Update node version limt in the @gitbeaker/node release to match the limit outlined by Got (>=10.19.0)

### ISSUE THREAD

**Issue by sdc224** [jdalrymple/gitbeaker#907]
Hi,

I am trying to install this project in my nodejs version `10.16.0`. It is failing due to ```got@11.3.0```.

Error logs of ```yarn add @gitbeaker/node```:
```bash
error got@11.3.0: The engine "node" is incompatible with this module. Expected version ">=10.19.0". Got "10.16.0"
error Found incompatible module.
```

Can't we use this library in `nodejs 10.16.0`? 

Actually I am in my organization laptop where I don't have admin permissions, so I am unable to update NodeJS here.

Please look into this, I am not giving this to Issue/Bug/Feature request, coz I am not sure what is the correct label for it.

**Comment by jdalrymple** [comment]
I should update the node limit on the @gitbeaker/node library sorry. To allow for the usage on earlier versions, i would have to revert to a version of got that was release back in February :/. You really should update your node version to the latest LTS.

**Comment by sdc224** [comment]
Hi @jdalrymple,

I have already mentioned why I can't update. So I tried `gitbeaker/browser` before this in my `electron` application, it was working fine for me till today. Now I have started reviewing some security aspects of `electron`.

When I was using ```nodeIntegration: true``` in electron, it was working fine. Now I have changed this flag to false, and added preload script, for fixing the security issue as mentioned by electron team.

After doing this, I am not able to run anything regarding `gitbeaker/browser`.

```bash
Module not found: Error: Can't resolve '@gitbeaker/browser' in 'D:\MyApp\electron-react-ts\app\gitlab'
```

```bash
!(function webpackMissingModule() { var e = new Error("Cannot find module '@gitbeaker/browser'"); e.code = 'MODULE_NOT_FOUND'; throw e;
```

That's why I am reverting here, if we can able to solve this problem, I don't need gitbeaker/node, which means I don't need to update nodejs😄

Please help me, it is very necessary, otherwise I need to continue with nodeIntegration: true😥

**Comment by jdalrymple** [comment]
I hear you. I'm not sure what to suggest :/ I've never come across that error before.

**Comment by sdc224** [comment]
Can you try to do the same by creating a sample electron app, if possible? Coz for me it is not working. Maybe you are the best person here to find out why webpack can't find out only this module(@gitbeaker/browser)

**Comment by jdalrymple** [comment]
I can try when i get some extra time, but it might be a while :(

**Comment by sdc224** [comment]
No Problem man! I will wait till then😊. Just let me know once you finish

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`23.4.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/23.4.0) :rocket:
